IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignment/Documentation.md

```diff
@@ -1,25 +1,78 @@
+
+The sequence of steps might change from case to case and might be repeated one or more times to achieve the economic objectives and fulfill regulatory safety requirements. 
+
+
+Contemporary alignment design almost always implements a 2.5 dimension approach. 
+
+The resulting and documented geometry might be very precise or just good enough to meet safety thresholds. This depends on factors like priorities of the management, date of the design - existing alignments might have been designed more then 50 years ago - or software tools used. Working with legacy data in a high precision BIM model requires a good understanding of these factors.
+
+**Distinction between business modeling and IFC core geometry**
+
+According to IFC modeling principles alignment entities are organised in two large parts. The two parts work together, but they can also be used independently from each other
+
+1. Business aspects of alignment
+2. Representation with the IFC geometry resources
+
+**Business aspects of alignment:** Here the focus is a on a schema structure as close to business terminology as possible. It is possible to have a very detailed segment structure with many domain specific properties attached. Examples for domain specific properties are design speed or cant deficiency.
+
+**Representation with IFC geometry resources:** Here the focus is on using as much of the established IFC geometry entities as possible. A mapping between Business aspects and IFC geometry is proposed.
+
+**IFC modelling**
   
-A single alignment may have:  
+In IFC a single alignment must have:  
   
-\* a horizontal alignment defined in the x/y plane of the engineering coordinate system  
-\* an accompanying vertical alignment, defined along the horizontal alignment in the distance along / z coordinate space  
-\* a relative alignment, defined as a span within another alignment and/or at constant or variable offsets  
-\* a 3D alignment, either computed from the horizontal and vertical alignment, or extracted from geospatial data.  
+* a horizontal alignment defined in the x/y plane of the engineering coordinate system  
+
+A single alignment may have: 
+
+* an accompanying vertical alignment, defined along the horizontal alignment in the distance along / z coordinate space  
+* a relative alignment, defined as a span within another alignment and/or at constant or variable offsets  
+* a 3D alignment, either computed from the horizontal and vertical alignment, or extracted from geospatial data.  
   
 Alignments may be aggregated into referents (_IfcReferent_) or derivative alignments. Derivative alignments may be used to indicate dependent alignments, such as an alignment for a bridge that is relative to a parent alignment for a road, where the child _IfcAlignment_ may have _Axis_ set to _IfcOffsetCurveByDistances_ that starts and ends at a span within the extent of the original _IfcAlignment_ defined at the _Axis_ of the parent _IfcAlignment_.  
```

Comment:
What does _Axis_ refer to here?

Reply:
This is previous text. Please provide a better description based on the final schema. @larswik @czapplitec ?

Reply:
New issue already created from one of the comments below.